linux-installer.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
-go build -o scientia-cli
+curl -s https://api.github.com/repos/goDoCer/scientiaCLI/releases/latest \
+    | grep "browser_download_url" \
+    | cut -d : -f 2,3 \
+    | tr -d \" \
+    | wget -O scientia-cli -qi -
```

Comment:
Can we add execution permissions automatically: `chmod +x ./scientia-cli` ?
```suggestion
    | wget -O scientia-cli -qi -
    | chmod +x ./scientia-cli
```

Reply:
not required, what we upload to the releases page is already an executable